### ISSUE TITLE
If checked is not passed into enhanced switch the aria-checked is missing

### ISSUE THREAD

**Issue by djbuckley** [manchesergit/material-ui#119]
if the checked property is not passed in to EnhancedSwitch , as in Checkbox, then the aria tag is not generated.
The aria-checked property should always be there for enhanced switch as it is a binary option.
